script/generate

```diff
@@ -4,7 +4,7 @@ require 'thor'
 
 require File.expand_path('../../generators/active_merchant_generator', __FILE__)
 
-Dir[File.expand_path('../..', __FILE__) + '/generators/*/*.rb'].each do |generator|
+Dir["#{File.expand_path('../..', __FILE__)}/generators/*/*.rb"].sort.each do |generator|
```

Comment:
I don't know if this matters since it is just sorting the array, and it is in a script but I have no idea why rubocop added the sort here. It's a strange thing to be picky about.